### ISSUE TITLE
Possible to limit number of documents returned?

### ISSUE THREAD

**Issue by leonpotgieter** [Elfocrash/Cosmonaut#19]
Hi, just wondering if there is any way one could limit the number of documents returned for a particular search query...  ie something like a .Take(10)

**Comment by Elfocrash** [comment]
You can do that by setting `MaxItemCount` on the `FeedOptions` parameter on the `Query` method to the numbers you wanna have returned.

Let me know if that solves your problem.

**Comment by leonpotgieter** [comment]
Hi, tx... that works perfectly. Much appreciated.

**Comment by leonpotgieter** [comment]
Apologies, spoke too soon... just realised in my test dataset that I only had the number of records I requested in my test. So no, I can't limit the number of records using feedoptions.

Tried the following.. but it still returns all records.

FeedOptions feedOptions = new FeedOptions() { MaxItemCount = 2};
var _users = await _userStore.Query(feedOptions).Where(x => x.Active).ToListAsync(cancellationToken);
